### PR TITLE
aws s3 storage for releases

### PR DESCRIPTION
**Pull request by metaclips**

### CODE REVIEW

File: .github/actions/aws_upload/action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    default: 'true'
+  aws_role_session_name:
+    description: AWS role session name
+    default: 'true'
+  aws_region:
+    description: AWS region
+    default: us-east-1
+  bucket_name:
+    description: AWS bucket name to upload assets
+    default: ockam_releases
```
Comment (snandam) [inline .github/actions/aws_upload/action.yml]: Isn't the bucket name `ockam-releases` ?

File: tools/scripts/release/release.sh [hunk 2]
```diff
@@ -30,6 +30,11 @@ if [[ -z $OWNER ]]; then
 fi
 release_name="release_$(date +'%d-%m-%Y')_$(date +'%s')"
 
+if [[ -z $OCKAM_RELEASE_URL ]]; then
+  echo "Please set Ockam AWS release URL e.g. https://ockam-release.s3.amazonaws.com"
```
Comment (snandam) [inline tools/scripts/release/release.sh]: should be `ockam-releases` ?

**Review by snandam — CHANGES_REQUESTED** [review]

File: tools/scripts/release/release.sh (hunk 2 above)
Comment (metaclips) [reply tools/scripts/release/release.sh]: Done. Thanks.

File: .github/actions/aws_upload/action.yml (hunk 1 above)
Comment (metaclips) [reply .github/actions/aws_upload/action.yml]: Done. Thanks.

**Review by snandam — APPROVED** [review]

**Pull request merged**